CHANGELOG.md

```diff
@@ -4,13 +4,39 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Common Changelog](https://common-changelog.org).
 
+[1.2.0]: https://github.com/sablier-labs/v2-periphery/compare/v1.1.1...v1.2.0
 [1.1.1]: https://github.com/sablier-labs/v2-periphery/compare/v1.1.0...v1.1.1
 [1.1.0]: https://github.com/sablier-labs/v2-periphery/compare/v1.0.3...v1.1.0
 [1.0.3]: https://github.com/sablier-labs/v2-periphery/compare/v1.0.2...v1.0.3
 [1.0.2]: https://github.com/sablier-labs/v2-periphery/compare/v1.0.1...v1.0.2
 [1.0.1]: https://github.com/sablier-labs/v2-periphery/compare/v1.0.0...v1.0.1
 [1.0.0]: https://github.com/sablier-labs/v2-periphery/releases/tag/v1.0.0
 
+## [1.2.0] - 2024-07-01
+
+### Changed
+
+- Rename `MerkleStreamer` to `MerkleLockup` ([#268](https://github.com/sablier-labs/v2-periphery/pull/268))
+- Rename `Batch` to `BatchLockup` ([#322](https://github.com/sablier-labs/v2-periphery/pull/322),
+  [#297](https://github.com/sablier-labs/v2-periphery/pull/297),
+  [#300](https://github.com/sablier-labs/v2-periphery/pull/300))
+- Refactor `Range` to `Timestamps` ([#335](https://github.com/sablier-labs/v2-periphery/pull/335))
+- Bump `sablier/v2-core` to v1.2.0
+- Bump build dependencies ([#351](https://github.com/sablier-labs/v2-periphery/pull/351),
+  [#283](https://github.com/sablier-labs/v2-periphery/pull/283))
+- Switch to Bun ([#249](https://github.com/sablier-labs/v2-periphery/pull/249))
+
```

Comment:
let's add:
 - Use Solidity v0.8.26 https://github.com/sablier-labs/v2-periphery/pull/351

Reply:
Its included in "Bump build dependencies ([#351](https://github.com/sablier-labs/v2-periphery/pull/351), [#283](https://github.com/sablier-labs/v2-periphery/pull/283))" section.

Reply:
but upgrading the solc version is not related to "dependencies"
`

Reply:
Understand. Then it should also be same in v2-core. I dont remember to mention solc version upgrade there.

Reply:
Feel free to push a commit to resolve these.

Reply:
i've already pushed a commit in core

Reply:
i will create a v1.2.0 github tag on core and the i will create another pr to bump the core which i will add to bump dependencies